lib/openid_connect/authorization/endpoint.rb

```diff
@@ -0,0 +1,37 @@
+module OpenidConnect
+  module Authorization
+    class Endpoint
+      attr_accessor :app, :user, :client, :redirect_uri, :response_type, :scopes, :_request_, :request_uri, :request_object
+      delegate :call, to: :app
+
+      def initialize(current_user)
+        @user = current_user
+        @app = Rack::OAuth2::Server::Authorize.new do |req, res|
+          build_attributes(req, res)
+          if OAuthApplication.available_response_types.include? Array(req.response_type).collect(&:to_s).join(' ')
```

Comment:
Prefer `map` over `collect`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.